### ISSUE TITLE
Unexpected keyword argument 'default_table' using tinydb 4.1.1

### ISSUE THREAD

**Issue by dagraham** [msiemens/tinydb#322]
#!/usr/bin/env python3.7
from tinydb import TinyDB
def initialize_tinydb(dbfile):
    return TinyDB(dbfile,
            default_table='items',
            indent=1, ensure_ascii=False)

if __name__ == '__main__':
   tinydb = initialize_tinydb("/home/dag/tinydb-test/db.json")
   tinydb.insert({'name': 'John', 'age': 22})
   print(f"tinydb: {tinydb}; {type(tinydb)}")

Under 4.1.1 this raises the exception: TypeError: __init__() got an unexpected keyword argument 'default_table' 
It runs fine without the default_table entry or under tinydb 3.15.2 with/without the default_table entry.

- Dan

**Comment by dagraham** [comment]
This fixes the problem for both 3.15.2 and 4.1.1:

    def initialize_tinydb(dbfile):
        if tinydb.__version__ >= '4.0.0':
            db = TinyDB(dbfile,
                      indent=1, ensure_ascii=False)
            db.default_table_name='items'
        else:
            db = TinyDB(dbfile,
                    default_table='items',
                    indent=1, ensure_ascii=False)
        return db

**Comment by farhan85** [comment]
There are some unit tests based on using multiple tables. They show that you can deal with the Table object directly:

https://github.com/msiemens/tinydb/blob/master/tests/test_tables.py

```python
db = tinydb.TinyDB(dbfile, indent=1, ensure_ascii=False)
items_table = db.table('items')
another_table = db.table('another_table_name')

items_table.insert({'name': 'John', 'age': 22})
another_table.insert({'foo': 'bar'})
```

If you do want to set default behaviour, the ChangeLog says you should now subclass `TinyDB`

https://tinydb.readthedocs.io/en/latest/changelog.html#breaking-changes

```python
class ItemsDB(TinyDB):
    # Alternatively, you could pass in the default name in __init__ and update this value dynamically
    default_table_name = 'items'
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

db = ItemsDB(dbfile)
```
